{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## For the Submitter
+Please make sure that you have followed all of the steps below, as
+it will help your PR be merged smoothly.
+
+### PR Cover Letter
+- [ ] Include A detailed description of the changes in the body of the PR
+
+### Testing
+- [ ] Unit, integration, acceptance tests as appropriate
+- [ ] Database migrations are backwards-compatible
+
+### DevOps assistance
+If it's needed, or even if you suspect it might be needed, do any/all these
```

Comment:
This section is also material for the developer to read and to then remove once they've obeyed it. It seems like too much text to be all caps though. Maybe there's another convention that would make this distinction clear.